android/fastlane/Fastfile

```diff
@@ -0,0 +1,152 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+    desc "Create new release"
+    lane :release do |options|
+        ensure_git_status_clean
+        ensure_git_branch( branch: 'main' )
+
+        options_release_number = options[:release_number]
+        options_next_dev_version = options[:next_dev_version]
+
+        newVersion = get_version_number(
+            release_number: options_release_number
+        )
+        nextSnapshot = get_next_development_version(
+            current_version: newVersion,
+            next_dev_version: options_next_dev_version
+        )
+
+
+        isInteractiveMode = options_release_number == nil
+
+        if !isInteractiveMode
+            UI.message("Release number: #{options_release_number}")
```

Comment:
If it is not interactive, seems like we are not doing anything?

Reply:
Yeah, will remove that part. Not useful right now